### ISSUE TITLE
Cannot import library on a server without X11

### ISSUE THREAD

**Issue by orestis** [matthewdowney/excel-clj#1]
Hi! Thanks for making this. It's the only Clojure excel library I could find that is based on a recent version of POI. 

Unfortunately, when I try to import the library on a server, I get an exception about X11 not being present, which I think is caused by the PDF stuff in there. I'll dig around to see if I can make a PR that pushes those to a separate namespace, so that I don't have to import them if I don't need them.

Thanks again!

**Comment by matthewdowney** [comment]
Hi there, you're welcome! And man that's an important limitation, thanks for bringing it up. I'd love to merge a PR fixing it!

**Comment by matthewdowney** [comment]
I was able to reproduce this locally in one of the Ubuntu TTYs. I ran `lein repl` in the excel-clj directory and then `(require '[excel-clj.core :as excel])` and it threw an X11 error (something to do with viz.clj).

Anyhow, adding the JVM flag `-Djava.awt.headless=true` (if you're using lein `:jvm-opts ["-Djava.awt.headless=true"]`) fixed it for me. I was able to create and write a workbook. 

Give it a shot and let me know if it fixes your issue!
